src/api/posts.ts

```diff
@@ -0,0 +1,25 @@
+import { Post } from '../react-app-env';
+
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
```suggestion
import  { BASE_URL } from ...
```
this constant already declared use it from api.ts file, EVERYWHERE